README.md

```diff
@@ -198,16 +227,29 @@ Thus, an example for a CLDF dataset description could look as follows:
 ### CLDF Data files
 
 While it is possible to add any kind of CSV files to a CLDF dataset, the CLDF standard
-recognizes (and attaches specified semantics) to tables described with a common property `dc:conformsTo` with one of the [table type](#cldf-components) URIs of the [CLDF ontology](http://cldf.clld.org/v1.0/terms.rdf) as value.
+recognizes (and attaches specified semantics) to tables described with a common property `dc:conformsTo` with one of 
+the [table type](#cldf-components) URIs of the [CLDF ontology](http://cldf.clld.org/v1.0/terms.rdf) as value.
 
 Additionally, CLDF semantics can be assigned to individual columns by 
 assigning one of the property URIs defined in the 
 [CLDF ontology](http://cldf.clld.org/v1.0/terms.rdf) as `propertyUrl`.
 
-Note: CLDF column properties are assumed to have a complete row (or rather the
-entity a row stores data about) as scope; e.g. a [source column](#sources)
-is assumed to provide source information for any piece of data in the row.
-Thus, each property can be used only once per table, which makes processing simpler.
+
+#### Column specifications
+
+- CLDF column properties are assumed to have a complete row (or rather the
+  entity a row stores data about) as scope; e.g. a [source column](#sources)
+  is assumed to provide source information for any piece of data in the row.
+  Thus, **each property can be used only once per table**, which makes processing simpler.
+- More generally, CLDF assumes **column names in a table to be unique**.
+- Since CLDF is designed to enable data reuse, data creators should assume that schema information
+  like table or column names ends up in all sorts of environments, e.g. as names in SQL databases or
+  as parts of URLs of a web application. Thus, it is recommended to stick to ASCII characters in such
+  names and avoid usage of punctuation other than `:._-`.
+- CSVW allows specifying columns as "list-valued", i.e. as containing a list of values (of the same datatype),
+  using the `separator` property. Thus, CLDF consumers **must** consult a column's `separator` property,
+  to figure out whether the value must be interpreted as list or not. Note that this also applies to foreign keys.
+  (Some properties in the CLDF Ontology require the respective column to be list-valued, though.)
```

Comment:
> (Some properties in the CLDF Ontology require the respective column to be list-valued, though.)

Should the CLDF Ontology not be also allowed to require the respective column to be non-list-valued?

Reply:
Yes, probably. Will try to assess that with #106